frontend/components/bookmarks/bookmarkComponent.tsx

```diff
@@ -0,0 +1,237 @@
+import { useEffect, useState } from "react";
```

Comment:
bookmarkComponent.tsx should have a more descriptive name like :  CreateBookmarkModule